app/templates/src/app.js

```diff
@@ -1,18 +1,16 @@
-import Map from "can/map/";
-import route from "can/route/";
-import 'can/map/define/';
-import 'can/route/pushstate/';
+import DefineMap from 'can-define/map/';
+import route from 'can-route';
+import 'can-route-pushstate';
 
-const AppViewModel = Map.extend({
-  define: {
-    message: {
-      value: 'Hello World!',
-      serialize: false
-    },
-    title: {
-      value: '<%= name %>',
-      serialize: false
-    }
+const AppViewModel = DefineMap.extend({
+  route: "string",
```

Comment:
Question: Can you explain why you added `route: "string"` to the `AppViewModel`?